components/ten_years_ago_component.py

```
@@ -0,0 +1,37 @@
+    for i in df:
+        # st.caption(str(i['id']) + ":" + i['category'])
+        with st.expander(i['title_summary']):
+            st.markdown("[" + i['body_summary'] + "](" + i['url'] + ")")
+
+def sidebarCon(df):
+    with st.sidebar:
+        with st.container(border=True):
+            st.info("✅ 10년 전 오늘")
+            with st.container(border=True):
+                for i in df:
+                    st.markdown(str(i['category']) + " : [" + i['title_summary'][:18] + "..." + "](" + i['url'] + ")")
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)